### ISSUE TITLE
Cache Config / Removing remote node manually

### ISSUE THREAD

**Issue by tsomeq** [munin-monitoring/munin#850]
I am using new preview version munin 2.999.

The munin-update updates RRD files with reading configurations from remote nodes.
Sometimes our remote server cannot respond to that request because it is very busy.

Previous version of munin stores the "NaN" value in RRD file, and graph is displayed with vertical blank part.
Graph of busy server is most interesting for me, but munin-httpd hides graph for these No-response servers.
So providing way to keep remote-configurations is greatly needed .

**Comment by niclan** [comment]
Non-responding hosts that are still configured on the master are supposed to be "sticky", e.g. keep linking and displaying the graphs.
